docs/docs-content/tutorials/edge/deploy-cluster.md

```diff
@@ -96,6 +96,13 @@ To complete this tutorial, you will need the following:
   [Create Registration Token](../../clusters/edge/site-deployment/site-installation/create-registration-token.md) guide.
   Copy the newly created token to a clipboard or notepad file to use later in this tutorial.
 
+  :::warning
+
+  Ensure that you set **Default Project** for the token. Otherwise, your Edge hosts will not register themselves with
```

Comment:
```suggestion
  Ensure that you set a default project for the token. Otherwise, your Edge hosts will not register themselves with
```
I know you are using bold because of the UI form element, but here you are more referring to the concept of default project generally

Reply:
Can you add this warning to the Create Registration Token page too?